hoodie-client/src/test/java/com/uber/hoodie/io/TestHoodieCompactor.java

```diff
@@ -138,6 +138,8 @@ public void testLogFileCountsAfterCompaction() throws Exception {
         HoodieWriteConfig config = getConfig();
         HoodieWriteClient writeClient = new HoodieWriteClient(jsc, config);
         String newCommitTime = "100";
+        writeClient.startCommitWithTime(newCommitTime);
```

Comment:
why are these needed? to force log the metadata?

Reply:
Yes, essentially, need the inflight file and metadata to be created.